compiler/src/dotty/tools/dotc/core/Contexts.scala

```diff
@@ -52,8 +52,10 @@ object Contexts {
   private val (notNullInfosLoc,     store8) = store7.newLocation[List[NotNullInfo]]()
   private val (importInfoLoc,       store9) = store8.newLocation[ImportInfo | Null]()
   private val (typeAssignerLoc,    store10) = store9.newLocation[TypeAssigner](TypeAssigner)
+  private val (usagesLoc,          store11) = store10.newLocation[Any]()          // unusages feature
+  private val (deferredChecksLoc,  store12) = store11.newLocation[DeferredChecks]()
 
-  private val initialStore = store10
+  private val initialStore = store12
```

Comment:
Adding something to the store is an expensive operation. It charges every time we copy a store with two additional slots to copy. if we look up the property not very often it's better to use a Property.